Client/Ecosia/Experiments/Unleash/SeedCounterNTPExperiment.swift

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Core
+
+struct SeedCounterNTPExperiment {
+    private enum Variant: String {
+        case control
+        case test
+    }
+    
+    private init() {}
+    
+    static var isEnabled: Bool {
+        Unleash.isEnabled(.seedCounterNTP) && variant != .control
+    }
+    
+    static private var variant: Variant {
+        Variant(rawValue: Unleash.getVariant(.seedCounterNTP).name) ?? .control
+    }
+    
+    // MARK: Analytics
+
+    /// Send onboarding card view analytics event, but just the first time it's called.
```

Comment:
I'm guessing a copy paste leftover comment from onboarding card? 🙈

Reply:
You guessed it right 😅 . Thanks for spotting 😛